rfcs/shared/build-system/triage-automation.md

```diff
@@ -0,0 +1,161 @@
+-->
+
+## Detailed Design or Proposal
+
+<!-- This is the bulk of the RFC. Explain the proposal or design in enough detail for the inteded audience to understand. -->
+
+After we [introduced new github issues beta](https://github.com/microsoft/fluentui/pull/22337) for reporting issues and feature request, the experience for both contributors and shield devs was significantly improved. A followup that was discussed is to implement further automation by leveraging Github Actions, This document will describe the approach in better detail.
+
+We would like to automate following steps:
+
+1. project label assignment
+2. package label assignment
+3. assign issue to project board/team
+
+**Pre-requirements:**
+
+- normalization/unification of labels within monorepo for projects
+
+_Current state:_
+
+- 2 labels for v0
+  - https://github.com/microsoft/fluentui/labels?q=northstar
+- 2 labels for v9
+  - https://github.com/microsoft/fluentui/labels/Fluent%20UI%20react-components%20%28v9%29
+  - https://github.com/microsoft/fluentui/labels/Fluent%20UI%20vNext
+- per component labels
+  - same labels mixed for various fluent version https://github.com/microsoft/fluentui/labels/Component%3A%20Dialog
+
+_Expected state:_
+
+- 1 label for v0
+- 1 label for v9
+- label per published package
+- "component labels" can stay as are but should be used solely for v8 (internal react package "sub-packages" https://github.com/microsoft/fluentui/labels/Component%3A%20Dialog)
+
+### 1. project label assignment
+
+> Note:
+>
+> - by "project" we mean one of libraries that exist within monorepo (v0,v8,v9,web-components)
+
+**Workflow:**
+
+Based on selected project within issue template, we should add appropriate label.
+
+<img width="1108" alt="image" src="https://user-images.githubusercontent.com/1223799/190106473-fd007d87-f09a-4493-aafd-781a8404ae45.png">
+
+> NOTE:
+>
+> - this is already implement via [github action](https://github.com/microsoft/fluentui/pull/24788)
+
+### 2. package label assignment
+
+> Note:
+>
+> - by "package" we mean official package name used to publish to npm registry without `@scope` prefix
+
+**Workflow:**
+
+Based on selected project within issue template, additional select should be dynamically generated so contributor can pick proper package.
+
+<img width="569" alt="example of flow" src="https://user-images.githubusercontent.com/1223799/190431945-56fcd343-6eb7-4145-b2b2-62f5919b8c31.png">
+
+Unfortunately **[dynamic input controls are not possible with github issues beta](https://github.com/community/community/discussions/29067)** which leaves us with following options how to implement this:
+
+- use only package names in "Library" picker
+  - cons
+    - huge list because we have quite a lot packages
+  - pros
+    - only 1 issue/feature template that we need to maintain
+    - removes need of `1. project label assignment` as labeling would be handled in one step
+- creating feature/bug templates per library
```

Comment:
I vote for template per library.
Simple to do while allowing us to have each template suit the library.

Reply:
updated https://github.com/microsoft/fluentui/pull/24817/commits/9117d9bd3b4582a016615d6e5c600573015161f5